contracts/Blockbin.sol

```diff
@@ -15,6 +15,10 @@ contract Blockbin {
         bytes data;
     }
 
+    function Blockbin() {
```

Comment:
Why is that needed?

Reply:
I believe it was needed when creating the contract object using web3. I added this a while back though, so it might now be necessary (I added it when playing around in the console). I'll try deploying and instantiating everything without this constructor and see if it still works.